.gitignore

```diff
@@ -1,37 +0,0 @@
-# OS X
```

Comment:
I wasn't looking to remove the `.gitignore` completely 😮 Just that line you added!